### PR TITLE
docs: Directionality in Oryx

### PR DESCRIPTION
**Pull request by tobi-or-not-tobi**
closes [HRZ-2167](https://spryker.atlassian.net/browse/HRZ-2167)

[HRZ-2167]: https://spryker.atlassian.net/browse/HRZ-2167?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/scos/dev/front-end-development/202307.0/oryx/building-applications/styling/oryx-icon-system.md [hunk 1]
```diff
@@ -146,6 +147,24 @@ export const app = appBuilder().withTheme({
...
+
+Oryx supports [directionality](/docs/scos/dev/front-end-development/{{page.version}}/oryx/internationalisation/directionality.html) and to ensure that some icons will flip accordingly, the icon style must be set with the `direction: true` config. The following example shows how the forward icon is configured to flip in RTL mode.
+
+```ts
+export const materialDesignIcons: IconMapper = {
+  id: "material-icons",
+  mapping: {
+    [IconTypes.Forward]: {
+      text: "chevron_right",
+      styles: { direction: true },
```
Comment (tolerants) [inline docs/scos/dev/front-end-development/202307.0/oryx/building-applications/styling/oryx-icon-system.md:162]: Shouldn't it be `direction: rtl`?

Comment (tolerants) [reply docs/scos/dev/front-end-development/202307.0/oryx/building-applications/styling/oryx-icon-system.md:162]: Hm, checking an implementation, but it looks a bit weird, we are using `styles`, property which should be used for css only, but passing some configuration.

File: docs/scos/dev/front-end-development/202307.0/oryx/internationalization/oryx-directionality.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+5. Icons: Icons that require flipping, such as arrows or directional indicators, are automatically flipped based on the selected direction. This ensures that icons are displayed correctly and in the appropriate orientation. The flipping of icons is done in CSS and is based on the `:dir(rtl)` [pseudo selector](https://caniuse.com/css-dir-pseudo). Since not all browsers have full support for this selector, this feature is a [progressive enhancement](https://developer.mozilla.org/en-US/docs/Glossary/Progressive_Enhancement).
+
+To enable directionality in Oryx, simply change the language to a locale like Arabic or Hebrew, and the direction will be automatically detected and applied. This feature provides flexibility and convenience for creating applications that support different languages and reading directions.
+
+{% info_block infoBox "Test RTL with labs package" %}
+
+To test RTL direction, the labs package have added a hardcoded "Arabic" locale.
```
Comment (tolerants) [inline docs/scos/dev/front-end-development/202307.0/oryx/internationalization/oryx-directionality.md:33]: Maybe a link to `labs` package?

File: docs/scos/dev/front-end-development/202307.0/oryx/building-applications/styling/oryx-icon-system.md (hunk 1 above)
Comment (tobi-or-not-tobi) [reply docs/scos/dev/front-end-development/202307.0/oryx/building-applications/styling/oryx-icon-system.md:162]: These are actually style configurations, such as font, fill, weight

```ts

export interface IconStyles {
  font?: string;
  fill?: number;
  weight?: number;
  grad?: number;
  optical?: number;
  size?: 'string';
  direction?: boolean;
}
```

Not sure why you think its confusing.

Comment (tolerants) [reply docs/scos/dev/front-end-development/202307.0/oryx/building-applications/styling/oryx-icon-system.md:162]: For me it was a bit confusing we are using other properties in styles object as css properties, but this one - in a different way. And I was not sure just from configuration what does it mean `direction: true` or `direction: false`. And there is no explanation in interface.

Comment (tobi-or-not-tobi) [reply docs/scos/dev/front-end-development/202307.0/oryx/building-applications/styling/oryx-icon-system.md:164]: understood. I've added an additional explanation to the parent section where these style configurations are introduced.

**Review by tolerants — APPROVED** [review]

File: docs/scos/dev/front-end-development/202307.0/oryx/internationalization/oryx-directionality.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Directionality in Oryx: Supporting RTL and LTR for applications"
+description: Learn how Oryx supports both right-to-left (RTL) and left-to-right (LTR) directionality. Discover how this feature automatically adjusts layouts, styling, and icons based on the selected direction, enabling developers to create culturally appropriate and accessible user interfaces for global applications.
+last_updated: November 6, 2023
+template: concept-topic-template
+---
+
+Directionality refers to the writing direction of a language, which can be either left-to-right (LTR) or right-to-left (RTL). LTR languages, such as English, are written from left to right, while RTL languages, such as Arabic, are written from right to left. Directionality affects the layout, styling, and alignment of text and other elements in a user interface.
+
+The web platform supports directionality natively by using the following techniques:
```
Comment (andriitserkovnyi) [inline docs/scos/dev/front-end-development/202307.0/oryx/internationalization/oryx-directionality.md:10]: By `web platform` do you mean Oryx?

**Pull request merged**